src/main/scala/iog/psg/cardano/experimental/cli/model/Policy.scala

```diff
@@ -46,7 +32,7 @@ object Policy {
   sealed trait Script
   object Script {
     case class Signature(keyHash: KeyHash[_ <: KeyType]) extends Script
-    case class Bound(slot: Int, after: Boolean) extends Script
+    case class Bound(slot: Long, after: Boolean) extends Script
```

Comment:
Maybe go up to BigInteger?

Reply:
The type is defined as DWORD64 so Long should be enough. ... 

https://input-output-hk.github.io/ouroboros-network/cardano-slotting/Cardano-Slotting-Slot.html#t:SlotNo